lib/config.js

```diff
@@ -48,7 +48,12 @@ config.mongodb.connectOptions = {
     journal: true
   },
   server: {
-    auto_reconnect: true
+    socketOptions: {
+      autoReconnect: true, // reconnect on connection failure
+      keepAlive: 15000, // TCP keep-alive every 15 seconds (useful for VPN'd connections)
+    },
+    reconnectInterval: 1000, // attempt reconnect every second
+    reconnectTries: 604800, // attempt to reconnect for a week and then give up
```

Comment:
I guess this is ok. Hopefully database monitoring tools will pick this up and send alerts since a monitor on the app won't get an alert (maybe) until the app quits a week later.